src/components/CurrentSponsors.tsx

```
@@ -0,0 +1,34 @@
+"use client"
+import * as React from "react"
+import Image from "next/image"
+
+import StreamLogo from "@/assets/stream.png"
+import CodeRabbitLogo from "@/assets/coderabbit.svg"
+import { Heart } from "lucide-react"
+import { Link } from "next-view-transitions"
+
+interface Props {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)